themes/gfsc/layouts/index.html.html

```diff
@@ -37,8 +37,8 @@ <h2 class="homepage__themes__title">{{ .Title }}</h2>
           <h3 class="window__header">{{ .Title }}</h3>
           <p class="window__content">{{ .Description }}</p>
           <div class="window__footer">
-            <div class="">
-              <a class="btn" href="{{ .Permalink }}">Browse</a>
+            <div class="window__footer__buttonbox">
```

Comment:
just a little pointer that with BEM there should never be nested elements like this - youre not meant to be copying the DOM youre meant to be making individual reusable components. so in this case it might simply be a new `buttonbox` namespace. this sort of thing is a CSS code smell for me that indicates i am probably nesting my CSS too deep - HTH

Reply:
The CSS in this repo is kind of BEM aesthetic, I'm trying to pull bits out of it's nested state as I touch stuff but unless we do a whole site refactor it's gonna be like this for a long time.

Reply:
That is understandable. I was responding to alfie's suggestion.

Seems like we both use BEM and non-BEM in places. I could fix up the class itself but I don't know how that will impact any other styling choices.

Reply:
totally fair if this is pre-existing - would prefer to refactor as we go but not if its a nightmare!